M2/Macaulay2/packages/FastLinAlg.m2

```diff
@@ -15,14 +15,14 @@ Version => "0.21", Date => "May 25th, 2020", Authors => {
     Email=> "yuhuiyao4ever@gmail.com"
     }
 }, --this file is in the public domain
-Headline => "A package for faster linear algebra operations.")
+Headline => "A package for faster linear algebra operations.", PackageExports => {"RandomRationalPoints"}, DebuggingMode => false, Reload=>false)
 export{
 --  "selectSmallestTerms",
   "chooseSubmatrixSmallestDegree", --there are checks
   "chooseSubmatrixLargestDegree", --there are checks
   "chooseRandomSubmatrix",
   "reorderPolynomialRing", --there are checks
-  "regularInCodimension", --there are checks
+  "Rn", --there are checks
```

Comment:
I've changed it and will push it shortly.  But how do I add an `Rn` as a synonym?
```
regularInCodimension.synonym = "Rn"
```
gives me an error
```
error: expected a list, hash table, database, or dictionary
```
Likewise `regularInCodimension.synoym = Rn`

Reply:
You would put both  `"regularInCodimension"` and  `"Rn" => "regularInCodimension"` in the list of exports.  

But, hold our horses: the name of the function is "R", not "Rn", because "n" is the name of the argument.  

We wouldn't use `sineTheta` to denote the sine function, would we?

Reply:
I've left it alone.